### DISCUSSION TITLE
Is this ThemeUiTab still working in Keycloak 25.0.1

### DISCUSSION THREAD

**Discussion by mihirvadalia** [dasniko/keycloak-extensions-demo#19] [Q&A]
When I try to use https://github.com/dasniko/keycloak-extensions-demo/blob/main/admin-ui/src/main/java/dasniko/keycloak/adminui/ThemeUiTab.java with Keycloak v25.0.1, it is not showing any new tab on Realm Settings Page.

In case anyone know, how to run this in Keycloak 25, help me with that.

Ping @dasniko for expert view on this.

**Comment by wjansoone** [comment]
Hello,
Not sure if it will solve your problem, but I had to enable the feature declarative-ui to show the custom tab. I started the server in development mode: kc start-dev --features=declarative-ui.
Mrs. Wilke Jansoone

**Comment by dasniko** [comment]
Reading the official docs helps...
